config/scorecard/patches/kuttl.config.yaml

```diff
@@ -1,7 +1,12 @@
 - op: add
   path: /stages/0/tests/-
   value:
+    entrypoint:
+    - rhoam-scorecard-tests
+    - happy-path
     image: quay.io/operator-framework/scorecard-test-kuttl:v2.0.0
     labels:
-      suite: rhoam-install
-      test: rhoam-install
\ No newline at end of file
+      cluster-product: ocp
+      cluster-size: small
+      phase: msp-main
+      test: happy-path
```

Comment:
updated according to [this config](https://gitlab.cee.redhat.com/cpaas-midstream/test-containers-ms/-/merge_requests/187/diffs)